code/io/joy_ff-sdl.cpp

```diff
@@ -727,7 +727,7 @@ void joy_ff_afterburn_on()
 		SDL_HapticStopEffect(haptic, pAfterburn2.id);
 
 		pAfterburn2.eff.periodic.length = SDL_HAPTIC_INFINITY;
-		pAfterburn2.eff.periodic.magnitude = 0x3fff;
+		pAfterburn2.eff.periodic.magnitude = 0x1999;
```

Comment:
The original code uses `INFINITE` for both effects while initializing so it was probably a mistake made while porting the code.